### PR TITLE
feat(cli): added `eggs upgrade` command

### LINKED ISSUE #78: [FEATURE] `eggs upgrade` to upgrade to the latest verison of eggs cli
**Is your feature request related to a problem? If so, please describe.**
N/A

**Describe the solution you'd like**
```sh
deno install --unstable --A -f -n eggs <install-path>
```
Notice the `-f` to force install the new version. 

**Describe alternatives you've considered**
N/A

**Additional context**
Simply run the above command as Deno.run. Could I be assigned to this?

### PR DESCRIPTION
**Pull request by divy-work**
Added `eggs upgrade` command to eggs cli.

__Working__:
1) Get latest version from Github's `version.ts`.
2) Executes `deno install`.

Closes #78

### CODE REVIEW

**Comment by oganexon** [comment]
I'm working on a module to notify users when there is an update for one of the CLIs they would have installed with eggs.  I think this can go hand in hand

File: eggs/src/deps.ts [hunk 1]
```diff
@@ -6,4 +6,5 @@ export { existsSync, expandGlobSync, writeJson } from "https://deno.land/std/fs/
 export { bold, green, yellow, red } from "https://deno.land/std/fmt/colors.ts";
 export { assertEquals } from "https://deno.land/std@v0.56.0/testing/asserts.ts";
 export * as base64 from "https://denopkg.com/chiefbiiko/base64/mod.ts";
+export { version } from "https://raw.githubusercontent.com/nestlandofficial/nest.land/master/eggs/src/version.ts";
```
Comment (tbaumer22) [inline eggs/src/deps.ts:9]: We'll want to store this data on the blockchain eventually, but for now, this should be fine.

**Comment by tbaumer22** [comment]
The code looks great, but we need to add documentation for this command before it can be merged.

**Comment by divy-work** [comment]
@tbaumer22 Looks like my `yarn` is playing with me. I'm unable to serve the website(getting bunch of errors). Cou you take a looks at the updated docs?

**Comment by tbaumer22** [comment]
Will do. Thanks!

**Comment by tbaumer22** [comment]
Going to merge this but will make a new PR with updated docs for this command. This command will be in our `0.1.4` release.

**Pull request merged**